justfile

```diff
@@ -38,7 +38,7 @@ erase-device +ARGS="nvs":
     cd device && espflash erase-parts --partition-table partitions.csv {{ARGS}}
 
 build-device BOARD=default_board +ARGS="":
-    cd device && cargo build --release --bin {{BOARD}} {{ARGS}}
+    cd device && cargo build --release --bin {{BOARD}} ${DEVICE_BUILD_ARGS:-} {{ARGS}}
```

Comment:
How do these `${DEVICE_BUILD_ARGS:-}` differ to just `{{ARGS}}`? Or is it to allow for setting via environment variables?

Reply:
Yep. Can set it via environment variable.